modules/claims_api/app/controllers/concerns/claims_api/v2/power_of_attorney_validation.rb

```diff
@@ -18,12 +18,22 @@ def validate_form_2122_and_2122a_submission_values(user_profile:, veteran_partic
 
       private
 
-      def feature_enabled_and_claimant_present?
-        Flipper.enabled?(:lighthouse_claims_api_poa_dependent_claimants) && form_attributes['claimant'].present?
+      def feature_enabled_and_claimant_present_and_not_self?
+        if Flipper.enabled?(:lighthouse_claims_api_poa_dependent_claimants) &&
+           form_attributes['claimant'].present?
+
+          relationship = form_attributes['claimant']['relationship']&.downcase
+          # even with both conditions being met
+          # this is the veteran if sent w/ relationship of 'Self'
+          # so we need to verify this is also true
+          relationship != 'self'
+        else
+          false
+        end
```

Comment:
Same suggestion as above. Is it worth centralizing this logic somewhere (to keep the code DRY)?

Reply:
Ya I agree, I was struggling to think of a good place for it between the two versions, but I will think on it and see what I can get

Reply:
broken that out into its own file, not sure how much, if at all, that may expand so it's own module _may_ be a bit much let me know what you think about the approach to unifying this method across version.